src/fare-contracts/components/ConsumeCarnetButton.tsx

```diff
@@ -0,0 +1,40 @@
+import {TicketValid} from '@atb/assets/svg/mono-icons/ticketing';
+import {useBottomSheet} from '@atb/components/bottom-sheet';
+import {LinkSectionItem, SectionItemProps} from '@atb/components/sections';
+import {ThemeIcon} from '@atb/components/theme-icon';
+import {useTheme} from '@atb/theme';
+import {FareContractTexts, useTranslation} from '@atb/translations';
+import React from 'react';
+import {ConsumeCarnetBottomSheet} from './ConsumeCarnetBottomSheet';
+
+type ConsumeCarnetButtonProps = SectionItemProps<{
```

Comment:
Should this be called `ConsumeCarnetSectionItemProps`?

Reply:
9665b1ce6fbdeba4e4aae8ba056467e3241ca58f

Reply:
Also note that I think just using `Props` is fine when it is not exported.